openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/CEEnvironmentProcessor.java

```diff
@@ -130,6 +147,21 @@ public void deleteOpenShiftResource(@Observes(precedence = -10) After event, Ope
     public void deleteEnvironment(@Observes(precedence = -10) AfterClass event, OpenShiftAdapter client,
         CubeOpenShiftConfiguration configuration) throws Exception {
         OpenShiftResourceFactory.deleteEnvironment(event.getTestClass(), client, configuration, classTemplateProcessor.getTemplates());
+        if (configuration.isNamespaceClassScopeEnabled()) {
+            namespaceService.get().delete(configuration.getNamespace() + "-" + event.getTestClass().getJavaClass().hashCode());
+        }
     }
 
+    private void createUniqueNamespace(CubeOpenShiftConfiguration configuration, String uniqueKey) {
+        namespace = configuration.getNamespace() + "-" + uniqueKey.toLowerCase().replaceAll("[^a-z0-9-]", "-");
+        if (namespace.chars().count() > 63) {
```

Comment:
Can we make `63` a constant with the name suggesting what does it really mean? Is that k8s limit?